warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -182,21 +185,23 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                 browser_inst = self.browser_object.open_browser(
                     browser_details["type"], webdriver_remote_url)
                 if browser_inst:
-                    browser_fullname = "{0}_{1}".format(system_name, browser_details["browser_name"])
+                    browser_fullname = "{0}_{1}".format(system_name,
+                                                        browser_details["browser_name"])
                     output_dict[browser_fullname] = browser_inst
-                    if "url" in browser_details and browser_details["url"]\
-                            is not None:
-                        result, url = self.browser_object.check_url(browser_details["url"])
-                        if result == True:
-                            result = self.browser_object.go_to(url,
-                                                               browser_inst)
+                    url = browser_details["url"]
+                    if url is not None:
+                        urlschema = urlparse(url)
+                        if urlschema.scheme:
+                            result = self.browser_object.go_to(url, browser_inst)
+                        else:
+                            status = False
```

Comment:
Hi @brettleehari,

Instead of setting the status to False in line 197, the result should be set to False - eventually in line 206 the status will become False anyway because we are and-ing result and status there.

result refers to a single iteration the for loop (each browser opened) while status is the status of the entire kw.

Thanks.